### DISCUSSION TITLE
How to read a history 'aka' retrun value of AutoModel.fit() or TextClassifier in autokeras?

### DISCUSSION THREAD

**Discussion by trisgelar** [keras-team/autokeras#1389] [Q&A]
Hi, I want to create simple graph to see the overfitting of model based on record of training loss values and metrics values on each epoch.
in tensorflow or keras i can get this kind information from resulting value of Model.fit function based on history attribute.
however, in autokeras to get this i think it would be analog using fit function, but, my history variable is empty.

I still confuse, about this, how to get record of training loss values and metrics values in autokeras?

Thanks.

**Comment by haifeng-jin** [comment]
Unfortunately, autokeras cannot support it now.
What do you think would be a good way to get the history?
If it make sense, we can consider implement it.
We are having multiple models trained, maybe the fit function should return the best model's history?

One way you can do now is to use TensorBoard, we can support that in the callbacks in fit.

**Reply by mjahmad** [reply]
This issues has been haunting me for sometime. It seems that AutoKeras still does not support history like Keras does.
